continuous_integration/environment-3.10.yaml

```diff
@@ -7,8 +7,8 @@ dependencies:
   # required dependencies
   - python=3.10
   - packaging
-  - numpy
-  - pandas
+  - numpy=1.23
```

Comment:
Because it will _probably_ just bring in the latest `numpy` release. Similar to how we test a few specific `X.Y` releases of `pandas` across CI builds, we do the same for `numpy` as a best effort approach